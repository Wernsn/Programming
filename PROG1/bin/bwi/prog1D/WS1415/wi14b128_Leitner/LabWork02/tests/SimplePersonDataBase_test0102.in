2
first
last
2014
1
1
1
0
