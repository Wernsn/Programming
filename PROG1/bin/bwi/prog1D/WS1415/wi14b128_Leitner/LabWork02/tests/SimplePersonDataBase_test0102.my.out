Welcome to the SimplePersonDatabase.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: 
ID  Lastname             Firstname            Birth
--------------------------------------------------------------------
001 last                 first                Wednesday, 01.01.2014
--------------------------------------------------------------------
1 Person(s) in database, 19 free.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: 
Thank you for using the SimplePersonDatabase.

